SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialLoadBalanceEnforcer.cpp

```diff
@@ -104,8 +80,10 @@ void SpatialLoadBalanceEnforcer::QueueAclAssignmentRequest(const Worker_EntityId
 	}
 }
 
-void SpatialLoadBalanceEnforcer::ProcessQueuedAclAssignmentRequests()
+TArray<SpatialLoadBalanceEnforcer::AclWriteAuthorityRequest> SpatialLoadBalanceEnforcer::ProcessQueuedAclAssignmentRequests()
 {
+	TArray<SpatialLoadBalanceEnforcer::AclWriteAuthorityRequest> PendingRequests;
```

Comment:
To avoid the copy, please pass this Array in as a reference, or MoveTemp it out.

Reply:
As I remember, Return-Value optimisation should take care of that automatically.